Shared/UserAgent.swift

```diff
@@ -98,10 +98,21 @@ public enum UserAgentPlatform {
 public struct CustomUserAgentConstant {
     private static let defaultMobileUA = UserAgentBuilder.defaultMobileUserAgent().userAgent()
     private static let customDesktopUA = UserAgentBuilder.defaultDesktopUserAgent().clone(extensions: "Version/\(AppInfo.appVersion) \(UserAgent.uaBitSafari)")
+    private static let mobileAdsUA = UserAgentBuilder.defaultMobileUserAgent().clone(extensions: "\(UserAgent.uaBitMobile) \(UserAgent.uaBitSafari)")
 
+    // when clicking the videos on nba.com they make requests to
+    // adobe.com, demdex.net ,doubleclick.net, googlesyndication.com, google.com, amazon-adsystem.com, rfihub.com
+    // and if the User Agent is different in most of them, the video on nba.com won't work either.
```

Comment:
According to this comment `adobe.com, `google.com` and `googlesyndication.com` are called from nba.com but all of these sites are not showing up on the list below. Do videos work without adjusting the user agent for them?

Reply:
when I tested at first I added all of them and then I removed the websites one by one until it didn't work anymore. It should work with the ones added in this PR but I have the list at the ready in case more will need to be introduced

Reply:
Then the comment should be updated and only mention the sites that block the video from working.

Reply:
ok, I'll update, made it like this initially so people can imply that those are not the only websites that the nba.com videos make requests from